binancial/data/get_trades_realtime.py

```
@@ -1,45 +1,95 @@
-def get_trades_realtime(stream, symbol, file_path):
+from typing import Any
+from datetime import datetime
+import logging
+import nest_asyncio
+from contextlib import contextmanager
```

Ruff (0.8.2)

`contextlib.contextmanager` imported but unused

Remove unused import: `contextlib.contextmanager`

(F401)